### PR TITLE
03-Bash-3: Add kernel statistics script

### PR DESCRIPTION
**Pull request by prosolvo4th**
kernel_stats.sh

### CODE REVIEW

File: 03-Bash/kernel_git/kernel_stats.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+# count *.c files
+declare -i C_COUNT
+declare -i CPP_COUNT
+declare -i PY_COUNT
+C_COUNT=0
+CPP_COUNT=0
+PY_COUNT=0
+
+for c_file in $(find "$1" -name "*.c" -type f)
```
Comment (yekovalyov) [inline 03-Bash/kernel_git/kernel_stats.sh]: Consider using  wc -l for calculate the quantity of lines.

File: 03-Bash/kernel_git/kernel_stats.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+fi
+
+# count *.c files
+declare -i C_COUNT
+declare -i CPP_COUNT
+declare -i PY_COUNT
+C_COUNT=0
+CPP_COUNT=0
+PY_COUNT=0
+
```
Comment (yekovalyov) [inline 03-Bash/kernel_git/kernel_stats.sh]: Are these lines required for updated version of script?

Comment (prosolvo4th) [reply 03-Bash/kernel_git/kernel_stats.sh]: It can be done without this variables

**Comment by prosolvo4th** [comment]
Can you point, what changes required?

File: 03-Bash/kernel_git/kernel_stats.sh (hunk 1 above)
Comment (prosolvo4th) [reply 03-Bash/kernel_git/kernel_stats.sh]: Done

File: 03-Bash/kernel_git/kernel_stats.sh (hunk 2 above)
Comment (prosolvo4th) [reply 03-Bash/kernel_git/kernel_stats.sh]: Variables removed

File: 03-Bash/kernel_git/kernel_stats.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+cd "$1"
+git checkout master 2&>1 /dev/null
+
+echo "revert commit count: $(git log --oneline | grep revert | wc -l)"
+
+git ls-files '*.py' | while read py_file
+do
+    echo "in file $py_file this autors :"
+    echo "$(git blame --line-porcelain $py_file | grep '^author ' | sed 's/^author //' | sort -f | uniq -c | sort -nr)"
```
Comment (yekovalyov) [inline 03-Bash/kernel_git/kernel_stats.sh]: This script gets statistic for each file separately. But statistic for all files together is required for this task.

Comment (prosolvo4th) [reply 03-Bash/kernel_git/kernel_stats.sh]: Done. Separated statistics collected in temporary file. Then uniq autors name counted by 'uniq'.

**Pull request merged**